### ISSUE TITLE
Expose ref for `<input>` element

### ISSUE THREAD

**Issue by justrhysism** [pqina/react-filepond#163]
**Is your feature request related to a problem? Please describe.**
Currently, it's not possible to get the `ref` for the underlying `<input>` element which makes this library difficult to use with libraries like "react-hook-form".

**Describe the solution you'd like**
Expose a prop `inputRef` so the author can provide their own ref.

**Describe alternatives you've considered**
Hacky solutions of traversing the DOM 😢 

**Additional context**
N/A

**Comment by rikschennink** [comment]
FilePond generates a custom list of input elements to store its data, the elements are removed when files are deleted, not sure how that would work with a `ref`

**Comment by bishnu-47** [comment]
how to get the file inside onSubmit then?

**Comment by rikschennink** [comment]
get a ref to FilePond component and call `getFiles()` on the FilePond instance in the ref.

**Comment by CestDiego** [comment]
I'm having an issue with this too because, even though we can submit by just querying the data. Now we don't get the awesome type checking that comes from using react-hook-form with a validator.

Regardless, thanks for working on this :D.

**Comment by rikschennink** [comment]
@CestDiego good point, I believe with the way I set up FilePond v5 this is addressed, so should be better in the future.
